frontend/cypress/integration/common/graph-pf.ts

```diff
@@ -4,16 +4,29 @@
 */
 
 import { Then } from '@badeball/cypress-cucumber-preprocessor';
+import { Controller, Edge, Visualization, Node, isNode, isEdge } from '@patternfly/react-topology';
 
 Then('user does not see a patternfly minigraph', () => {
   cy.get('#MiniGraphCard').find('h5').contains('Empty Graph');
 });
 
 Then('user sees a patternfly minigraph', () => {
```

Comment:
This step is used multiple times across the Cypress tests. Have you verified that it works for all use cases? The Cytoscape test is quite generic (it just checks if the #cytoscape-graph element is visible), but the PFT test performs additional checks on the number and names of the minigraph nodes. That also reminds me that we have to modify the PFT graph id from #cytoscape-graph to #pft-graph

Reply:
I tested it in the app, service and workload details page (for single cluster). 

I will test it for multi cluster, probably a change needs to be made for the multicluster part, but this draft PR is just to showcase how to get the nodes and edges and instrospect the graph, I will address mc part soon.

Reply:
It's true that the original was pretty simple, so I decided to at least add some assertions about the data.

Reply:
Thanks @leandroberetta for checking minigraph test in all scenarios. I agree that the more assertions the better :slightly_smiling_face: